src/pages/DirectMessage.tsx

```diff
@@ -148,95 +152,103 @@ const DirectMessage = () => {
-                placeholder="Send a message"
-                size="small"
-                value={textContent}
-              />
-            </Box>
-          </form>
+          <Box sx={styles.chatContainer}>
+            <List component="div" ref={messageRef} sx={styles.messageList}>
+              {messages.map((o) => (
+                <ListItem component="div" key={o.messageId}>
+                  <ListItemText
+                    sx={
+                      o.sentUserId === user.userId
+                        ? styles.sentMessage
+                        : styles.message
+                    }
+                    disableTypography
+                    primary={
+                      <Box
+                        sx={
+                          o.sentUserId === user.userId
+                            ? styles.sentMessageText
+                            : styles.messageText
+                        }
+                      >
+                        <Typography variant="body2">{o.textContent}</Typography>
+                      </Box>
+                    }
+                    secondary={
+                      <Typography sx={styles.timestamp} variant="caption">
+                        {o.timestamp}
+                      </Typography>
+                    }
+                  />
+                </ListItem>
+              ))}
+            </List>
+            <Divider />
+            <form onSubmit={onSubmit}>
+              <Box sx={styles.chatInputContainer}>
+                <TextField
+                  autoComplete="off"
+                  fullWidth
+                  hiddenLabel
+                  InputProps={{
+                    startAdornment: (
+                      <InputAdornment position="start">
+                        <IconButton>
+                          <AddPhotoAlternateOutlinedIcon />
+                        </IconButton>
+                        <IconButton>
+                          <EmojiEmotionsOutlinedIcon />
+                        </IconButton>
+                        <IconButton>
+                          <GifBoxOutlinedIcon />
+                        </IconButton>
+                      </InputAdornment>
+                    ),
+                    endAdornment: (
+                      <InputAdornment position="end">
+                        <IconButton
+                          type="submit"
+                          disabled={!textContent.trim()}
+                        >
+                          <SendIcon />
+                        </IconButton>
+                      </InputAdornment>
+                    ),
+                  }}
+                  onChange={(e) => setTextContent(e.target.value)}
+                  placeholder="Send a message"
+                  size="small"
+                  value={textContent}
+                />
+              </Box>
+            </form>
+          </Box>
         </Box>
       </Box>
```

Comment:
As discussed offline, there's some complication around this since it adds an extra pixel, which makes the layout of the messages pages 1 pixel off compared to the other pages. Hopefully we can solve this in another ticket